packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -476,26 +570,27 @@ class Slider extends React.Component {
         onClick={this.handleClick}
         onMouseDown={this.handleMouseDown}
         onTouchStartCapture={this.handleTouchStart}
-        onTouchMove={this.handleMouseMove}
+        onTouchMove={this.handleTouchMove}
         ref={ref => {
           this.containerRef = ReactDOM.findDOMNode(ref);
         }}
         {...other}
       >
         <div className={containerClasses}>
           <div className={trackBeforeClasses} style={inlineTrackBeforeStyles} />
-          <ButtonBase
-            className={thumbClasses}
-            disableRipple
-            style={inlineThumbStyles}
-            onBlur={this.handleBlur}
-            onKeyDown={this.handleKeyDown}
-            onTouchStartCapture={this.handleTouchStart}
-            onTouchMove={this.handleMouseMove}
-            onFocusVisible={this.handleFocus}
-          >
-            {ThumbIcon}
-          </ButtonBase>
+          <div className={thumbWrapperClasses} style={inlineThumbStyles}>
```

Comment:
Could you explain why we need this wrapper and not just merge `className` and `style` with the `ButtonBase`?

Reply:
Percentage in transform is relative to the element itself, not the parent as in other cases.
As I want `100%` to represent the end of the slider and not `12px` (the width of the thumb) I have to do the transform on an element that have the same size as the slider itself.